### PR TITLE
#18: Create Registration UI

### PR DESCRIPTION
**Pull request by ShashankKasera**

### CODE REVIEW

**Comment by ShashankKasera** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Having a proper description on PR gives more clarity to the reviewer. Please ensure that the PR has a neatly written description explaining the purpose of this change request.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Let's also assign this pull request to someone for review.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems">Danger Kotlin</a> against 23dc255385f27a2a06e59945b0c67aaf2f09afc5
</p>

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -15,16 +15,26 @@
...
             android:exported="true">
+<!--            <intent-filter>-->
+<!--                <action android:name="android.intent.action.MAIN" />-->
+
+<!--                <category android:name="android.intent.category.LAUNCHER" />-->
+<!--            </intent-filter>-->
+
+<!--            <meta-data-->
+<!--                android:name="android.app.lib_name"-->
+<!--                android:value="" />-->
```
Comment (mayankkasera) [inline app/src/main/AndroidManifest.xml:26]: Remove this unused comment.

Comment (ShashankKasera) [reply app/src/main/AndroidManifest.xml:26]: done

File: authentication/src/main/res/layout/activity_login.xml [hunk 2]
```diff
@@ -76,7 +76,7 @@
                 android:background="@null"
                 android:gravity="center"
                 android:lines="1"
-                android:hint="Enter Email"
+                android:hint="Email Address"
```
Comment (mayankkasera) [inline authentication/src/main/res/layout/activity_login.xml:79]: move this in strings.xml

File: authentication/src/main/res/layout/activity_registration.xml [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+        app:cardElevation="8dp"
+        android:background="@color/primary"
+        android:layout_marginTop="60dp"
+        android:layout_marginLeft="34dp"
+        android:layout_marginRight="34dp"
+        app:cardPreventCornerOverlap="false"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/iv_logo">
+        <!-- Your CardView content goes here -->
```
Comment (mayankkasera) [inline authentication/src/main/res/layout/activity_registration.xml:37]: remove this comment

File: authentication/src/main/res/layout/activity_registration.xml [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+                android:background="@null"
+                android:gravity="center"
+                android:lines="1"
+                android:hint="Your Name"
+                android:inputType="text"
+                android:textColorHint="@color/white"
+                app:layout_constraintStart_toEndOf="@+id/cv_User_logo" />
+        </androidx.constraintlayout.widget.ConstraintLayout>
+    </androidx.cardview.widget.CardView>
+    <androidx.cardview.widget.CardView
```
Comment (mayankkasera) [inline authentication/src/main/res/layout/activity_registration.xml:83]: format code.

File: authentication/src/main/res/layout/activity_registration.xml [hunk 5]
```diff
@@ -0,0 +1,283 @@
...
+            </androidx.cardview.widget.CardView>
+
+            <EditText
+                android:id="@+id/et_Email"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:background="@null"
+                android:gravity="center"
+                android:lines="1"
+                android:hint="Email Address"
```
Comment (mayankkasera) [inline authentication/src/main/res/layout/activity_registration.xml:138]: move all strings to strings.xml

File: authentication/src/main/res/layout/activity_login.xml (hunk 2 above)
Comment (ShashankKasera) [reply authentication/src/main/res/layout/activity_login.xml]: done

File: authentication/src/main/res/layout/activity_registration.xml (hunk 3 above)
Comment (ShashankKasera) [reply authentication/src/main/res/layout/activity_registration.xml:35]: done

File: authentication/src/main/res/layout/activity_registration.xml (hunk 4 above)
Comment (ShashankKasera) [reply authentication/src/main/res/layout/activity_registration.xml:82]: don

File: authentication/src/main/res/layout/activity_registration.xml (hunk 5 above)
Comment (ShashankKasera) [reply authentication/src/main/res/layout/activity_registration.xml]: done

**Pull request merged**